style.css

```diff
@@ -0,0 +1,733 @@
+  font-weight: 400px;
+  max-width: 880px;
+  min-width: 327px;
+  color: #344563;
+  line-height: 24px;
+  margin-bottom: 0.3em;
+}
+
+.homepage .connect,
+.about-myself .connect {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #7f8cff;
+  padding: 0.2em 0;
+  cursor: pointer;
+}
+
+.socialmedia span i {
+  font-size: 21px;
+  font-weight: bold;
+  color: #505f79;
+  margin: auto 0.4em;
+  cursor: pointer;
+}
+
+.socialmedia span i:hover {
+  font-weight: bolder;
+}
+
+/* smaller screen view  */
+@media screen and (max-width: 375px) {
+  .homepage h1 {
+    font-size: 35px;
+    min-width: 100px;
+  }
+
+  .homepage p {
+    font-size: 20px;
+    width: 80%;
+    min-width: 270px;
+  }
+
+  .homepage .connect {
+    font-size: 18px;
+  }
+}
+
+/* for desktop view  */
+@media screen and (min-width: 768px) {
+  header {
+    padding: 1em 6em;
+  }
+
+  header .menuicon {
+    display: none;
+  }
+
+  header .navbar {
+    display: inline-block;
+  }
+
+  .homepage {
+    background-color: #fff;
+    align-items: center;
+    padding-left: 15%;
+    padding-top: 10em;
+  }
+
+  .homepage h1 {
+    color: #091e42;
+  }
+}
+
+/* This is the work section  */
+.work {
+  font-family: 'Poppins', sans-serif;
+  background-color: #f7f7f9;
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-columns: repeat(1, 1fr);
+  justify-content: center;
+  align-content: center;
+  margin-left: 50%;
+  transform: translateX(-50%);
+  row-gap: 25px;
+}
+
+.work .card {
+  width: 327px;
+  background-color: #fff;
+  border: 1px solid rgba(0, 0, 0, 0.3);
+  border-radius: 20px;
+  margin-bottom: 2em;
+}
+
+.card .card-container {
+  width: 90%;
+  margin-left: 5%;
+}
+
+.work .card img {
+  width: 295px;
+  height: 230px;
+  background-color: skyblue;
+  border-radius: 8px;
+  margin: 1em 0;
+}
+
+.card-items h2 {
+  font-family: 'Poppins', sans-serif;
+  width: 70%;
+  font-size: 35px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #173b4d;
+  margin-bottom: 10px;
+}
+
+.card-items .work-details {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  font-size: 15px;
+  line-height: 16px;
+  text-align: left;
+  margin-bottom: 10px;
+}
+
+.card .work-details .name {
+  font-weight: 600;
+  color: #344563;
+}
+
+.work-details .circle {
+  font-weight: 300;
+  width: 10px;
+  height: 10px;
+  background-color: #7a869a;
+  border-radius: 50%;
+}
+
+.card-items .work-details span {
+  color: #7a869a;
+}
+
+.card-items p {
+  font-size: 17px;
+  font-weight: 400;
+  color: #344563;
+  line-height: 24px;
+  text-align: left;
+  margin-right: 10px;
+  margin-bottom: 10px;
+}
+
+.card-items .language {
+  display: flex;
+  list-style-type: none;
+  margin: 0.5em 0;
+  cursor: pointer;
+}
+
+.language li {
+  font-size: 14px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin: 5px 0;
+  cursor: pointer;
+}
+
+.card1 .language,
+.card2 .language {
+  width: 70%;
```

Comment:
Kindly consider removing the width property from this to make it look like the provided Figma design 😁. Since we are practicing our skills for transforming design into code, following the design as much as possible in this project would be important 💯.

![Screenshot 2023-11-25 115809](https://github.com/HsuMyatPhoo/PortfolioFinishMobileVersion/assets/102011195/df4ebc51-ef8c-41f0-8566-a69857de9665)